internal/router/controllers/controller.go

```diff
@@ -24,14 +29,18 @@ type SQLRunner interface {
 	RunReadQuery(ctx context.Context, stmt string) (*tableland.TableData, error)
 }
 
-// UserController defines the HTTP handlers for interacting with user tables.
-type UserController struct {
-	runner SQLRunner
+// Controller defines the HTTP handlers for interacting with user tables.
+type Controller struct {
+	runner        SQLRunner
+	systemService system.SystemService
 }
```

Comment:
Previous `UserController`, `SystemController` and `InfraController` where unified in `Controller`.